docs/services/devportal.md

```diff
@@ -0,0 +1,154 @@
+
+Complete the mandatory fields (marked with `*`) and make sure to give the application a unique name and select the number of requests per minute.
+When finished, click on the `SAVE` button.
+
+![Complete New App](../images/services/devportal-complete-new-app.png)
+
+!!! note
+    On your first login you will find the `DefaultApplication` which is created by default
+
+!!! note
+    The quota of requests per minute will be shared by all subscribers to the Application over all APIs
+
+### Configuring Production Keys
+
+Once the Application is created, create the Production Keys by clicking on `Production Keys` in the left panel:
+
+![FirecREST production keys](../images/services/devportal-keys.png)
+
+Click on the `Generate Keys` button at the bottom of the page to generate the production keys of your application:
+
+![FirecREST existing keys](../images/services/devportal-generate-keys.png)
+
+Once the keys are generated, you will see the pair `Consumer Key` (also known as "client id") and `Consumer Secret` (also known as "client secret").
+
+![FirecREST keys](../images/services/devportal-keys-overview.png)
+
+!!! note
+    At the moment, only FIRECREST keys are available for the users
+
+### Subscribe to an API
+
+Once you have set up your Application, is time to subscribe it to an API.
+
+To do so:
+
+* (8.a) click on the :fontawesome-solid-rss: `Subscriptions` option on the left panel
+* (8.b) click the :fontawesome-solid-circle-plus: `SUBSCRIBE APIS` button
+* (8.c) choose the [business plan][ref-devportal-api-info] and the version of the API you want to subscribe to by clicking the `SUBSCRIBE` button on the right-side of the requested API.
+
+![Devportal subscriptions](../images/services/devportal-api-subscriptions.png)
+
+## Manage your applications
+
+Back on the Subscription Management page, you can review your active subscriptions and APIs that your Application has access to.
+
+![Devportal subscription management](../images/services/devportal-api-subscriptions-management.png)
+
+### Regenerate production keys
+
+It's a good practice to rotate or regenerate the production keys of your application to increase security.
+
+To do this, go to the `Production Keys` view of the desired application and click on the button `REMOVE KEYS`:
+
+![Devportal reset keys](../images/services/devportal-regenerate-keys-01.png)
+
+![Devportal reset keys confirm](../images/services/devportal-regenerate-keys-02.png)
+
+To regenerate the keys, refer back to the [Configuring Production Keys](#configuring-production-keys) section.
+
+!!!info
+    After resetting the credentials, you will need to reconfigure your software(s) to use the new keys
+
+### Remove your application
+
+To delete your application (which means removing all subscriptions and production keys), refer to the Application view and click on the `DELETE` button in the top-right corner:
+
+![Devportal remove app](../images/services/devportal-remove-app.png)
```

Comment:
I think that this image might not be needed.

A better image, if you think one must be provided, would show just the top part of the screen before the "DELETE" button has been clicked. But `click on the DELETE button in the top-right corner` is probably enough.